HealthcareBase/Model/Users/Survey/SurveySection.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using HealthcareBase.Repository.Generics;
+
+namespace HealthcareBase.Model.Users.Survey
+{
+    public class SurveySection:IEntity<int>
+    {
+        [Key]public int Id { get; set; }
+        public string SectionName { get; set; }
+        public bool IsDoctorSection { get; set; }
```

Comment:
Should you use some kind of enumeration like SectionType?

Reply:
As long as section can be only doctor section or default section boolean satisfies our needs.
In future, if we find out that we need wider range of survey boolean will be replaced with a simple enumeration.